resources/views/forum/_post_toolbar.blade.php

```diff
@@ -2,7 +2,12 @@
     Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the GNU Affero General Public License v3.0.
     See the LICENCE file in the repository root for full licence text.
 --}}
-<div class="post-box-toolbar">
+<div
+    class="post-box-toolbar"
+    @if (isset($disabled) && $disabled === true)
+        disabled
```

Comment:
`div` doesn't have `disabled` attribute. This should go either as class name (as modifier perhaps) or data attribute.